Examples/Modules/qed/breit_wheeler/inputs_2d_tau_init

```diff
@@ -62,6 +62,32 @@ photons.uz_th = 100.
 ##########QED####################
 photons.do_qed = 1
 photons.do_qed_breit_wheeler = 1
+photons.qed_breit_wheeler_ele_product_species = ele_bw
+photons.qed_breit_wheeler_pos_product_species = pos_bw
+#################################
+
+### PRODUCT SPECIES ###
+ele_bw.charge = -q_e
+ele_bw.mass = m_e
+ele_bw.injection_style = nuniformpercell
+ele_bw.num_particles_per_cell_each_dim = 1 1
+ele_bw.profile = constant
+ele_bw.density = 0.0
+ele_bw.momentum_distribution_type = "gaussian"
+ele_bw.xmin = 1 ## Ugly trick to avoid electrons at T=0
+ele_bw.xmax = -1 ##  Ugly trick to avoid electrons at T=0
+ele_bw.do_qed = 0
+
+pos_bw.charge = q_e
+pos_bw.mass = m_e
+pos_bw.injection_style = nuniformpercell
+pos_bw.num_particles_per_cell_each_dim = 1 1
+pos_bw.profile = constant
+pos_bw.density = 0.0
+pos_bw.momentum_distribution_type = "gaussian"
+pos_bw.xmin = 1 ## Ugly trick to avoid positrons at T=0
+pos_bw.xmax = -1 ##  Ugly trick to avoid positrons at T=0
+pos_bw.do_qed = 0
```

Comment:
I really like what you propose (including not allowing to create a species with just `photons.qed_breit_wheeler_ele_product_species = ele_bw`)!
Probably the easiest way to achieve this could be to allow for default values for most of the parameters of a species (except `mass` *and* `charge` or `species_type`).  The default value for `profile` could be something like `empty` or `do_not_generate`, so that the code could know that methods for particle initialization should not be called. 
Would something like that be OK? In case, I could work on a dedicated PR...

Reply:
Yes, that sounds good! You are most welcome to look into it.